src/main/java/loqor/ait/client/data/ClientLandingManager.java

```diff
@@ -63,7 +63,7 @@ public void receive(PacketByteBuf buf) {
         ChunkPos chunkPos = buf.readChunkPos();
 
         if (action == LandingPadManager.Network.Action.ADD) {
-            this.regions.put(chunkPos, NetworkUtil.receive(LandingPadRegion.CODEC, buf));
+            this.regions.put(chunkPos, Network.receive(LandingPadRegion.CODEC, buf));
```

Comment:
It might be the matter of personal preferences buuut I think NetworkUtil should stay as NetworkUtil
I dont think you can instantiate it (although i may be completely wrong as i just started the review, ill probably come back to this later)

Reply:
yeah good idea..